lib/bloc/account/account_model.dart

```diff
@@ -403,15 +394,11 @@ class AccountModel {
   String validatePayment(Int64 amount, bool outgoing) {
     final texts = getSystemAppLocalizations();
     if (maxPaymentAmount != null && amount > maxPaymentAmount) {
-      return texts.valid_payment_error_exceeds_the_limit(
-        currency.format(maxPaymentAmount),
-      );
+      return texts.valid_payment_error_exceeds_the_limit("");
```

Comment:
There are better approaches than passing an empty string here.
Removing the variable from the text entry point, you can call the texts without arguments by calling `return texts.valid_payment_error_exceeds_the_limit`

Reply:
Regarding you feedback,I was not able to pass a null param so I ended up with generating a new variable without a param.

Is it possible to generate translations that are optional? 

If it is fact is not possible please see this pr https://github.com/breez/Breez-Translations/pull/7

Reply:
Reviewed; thanks for addressing the issue.